agent/cache/cache.go

```diff
@@ -35,6 +35,9 @@ import (
 const (
 	CacheRefreshBackoffMin = 3               // 3 attempts before backing off
 	CacheRefreshMaxWait    = 1 * time.Minute // maximum backoff wait time
+
+	defaultEntryFetchRate     = rate.Inf // don't rate limit
+	defaultEntryFetchMaxBurst = 1000     // this is well above anything that should be necessary
```

Comment:
should this value diverge from the defaults in `agent/config/builder.go`?

Reply:
I think if rate is `rate.Inf` the burst value is ignored, so we could use any value here.

Reply:
I went ahead and removed the constants from builder.go and just made builder.go reference the now exported consts from the cache package.

Reply:
ah, yep: `As a special case, if r == Inf (the infinite rate), b is ignored.`